src/middleware.js

```diff
@@ -36,5 +36,13 @@ export function middleware(request, event) {
 }
 
 export const config = {
-  matcher: '/writing/:path/'
-}
+  matcher: [
+    {
+      source: '/writing/:path/',
+      missing: [
+      { type: 'header', key: 'next-router-prefetch' },
+        { type: 'header', key: 'purpose', value: 'prefetch' },
+      ],
+    },
+  ],
+};
```

Comment:
```suggestion
  matcher: [
    {
      source: '/writing/:path/',
      missing: [
        { type: 'header', key: 'next-router-prefetch' },
        { type: 'header', key: 'purpose', value: 'prefetch' },
      ],
    },
  ],
};
```

Reply:
Very sorry, now i format this file in prettier, 
hope this time work.